tests/unit/source_code/samples/dfsa/create_cloud_files.sql

```diff
@@ -0,0 +1,13 @@
+-- Databricks notebook source
+CREATE OR REFRESH STREAMING LIVE TABLE pcmd_stream_bronze
+COMMENT "PCMD Stream - Bronze"
+AS SELECT *
+  FROM cloud_files(
+    "s3a://db-gtm-industry-solutions/data/CME/telco/PCMD",
```

Comment:
make these functional tests - e.g. `# ucx[direct-storage-access:+1:0:+1:23] ...`.

Reply:
The functional tests only support linting, so doing this now would significantly increase the changes in this PR. Can we do this in an ultimate PR once all related PRs are merged ?